tests/test_parameters.py

```diff
@@ -342,3 +344,68 @@ def test_parameter_from_owl_uri_extra_param(caplog):
                            units='d', # wrong parameter
                            name='example')
     assert "parameter units with value d not used to construct <class 'cdci_data_analysis.analysis.parameters.Time'>" in caplog.text
+
+@pytest.mark.fast
+def test_parameter_bounds():
```

Comment:
this is good, but maybe it'd really help to ask the live dispatcher fixture, to check what sort of response we are getting out all the way.

A test for oda_api could be helpful too.